### PR TITLE
feat(arb-referrals): change out arb referrals

### PR DESCRIPTION
**Pull request by james-a-morris**
This is WIP and is blocked at the moment. The main portion of this change is to modify static files.

### CODE REVIEW

**Comment by james-a-morris** [comment]
Blocked on Scraper + new design elements from Tim

**Comment by james-a-morris** [comment]
Blocked but need to publish PR so that I can change settings in Vercel for a custom ENV variable

**Review by amateima — COMMENTED** [review]
Screenshot 2024-06-14 at 19 27 20

I selected Optimism as destination chain but I don't see OP rewards being estimated. Is this sort of a regression?

**Comment by james-a-morris** [comment]
> a13037d

The most recent commit was stashed, pushed now

**Comment by james-a-morris** [comment]
> Screenshot 2024-06-14 at 19 27 20
> I selected Optimism as destination chain but I don't see OP rewards being estimated. Is this sort of a regression?

https://github.com/across-protocol/frontend-v3/pull/1063#issuecomment-2168373868

File: src/views/Bridge/components/EstimatedTable.tsx [hunk 1]
```diff
@@ -506,8 +506,23 @@ const ReferralRewardWrapper = styled.div<{
...
+          : "op-red-15"
+      ]};
+  background: ${({ rewardTokenSymbol }) =>
+    COLORS[
+      rewardTokenSymbol === "ACX"
+        ? "aqua-5"
+        : rewardTokenSymbol === "ARB"
+        ? "arb-blue-5"
+        : "op-red-5"
+    ]};
```
[the remark this answers is not in the document]
Comment (james-a-morris) [reply src/views/Bridge/components/EstimatedTable.tsx]: fair call!

**Pull request merged**